charts/pgadmin4/Chart.yaml

```diff
@@ -1,7 +1,7 @@
-apiVersion: v1
+apiVersion: v2
 description: pgAdmin4 is a web based administration tool for PostgreSQL database
 name: pgadmin4
-version: 1.38.0
+version: 1.39.0
```

Comment:
this should have been bumped to V2 because of the breaking changes

Reply:
I rolled back, this one:

> Renamed ingressClassName to className, to be more inline with other helm charts.

So sorry for the confusion!

You still think I need to push towards 2.0 because of k8s 1.19+? That is a really old version of Kubernetes. Almost no charts supports it anymore.